matchzoo/preprocessor/process_units.py

```diff
@@ -196,41 +203,15 @@ def _create_n_letters(self, tokens: list, ngram: int=3) -> list:
         :param ngram: By default use 3-gram (tri-letter).
 
         :return n_letters: generated n_letters.
-        :return: length of n_letters, dimensionality of :DSSMModel:.
         """
         n_letters = set()
         for token in tokens:
             token = '#' + token + '#'
             while len(token) >= ngram:
                 n_letters.add(token[:ngram])
                 token = token[1:]
-        return n_letters, len(n_letters)
-
-    def transform(self, tokens: list, ngram: int=3) -> list:
-        """
-        Transform token into tri-letter.
-
-        For example, `word` should be represented as `#wo`,
-        `wor`, `ord` and `rd#`.
-
-        :param tokens: list of tokens to be transformed.
-        :param ngram: By default use 3-gram (tri-letter).
-
-        :return: set of tri-letters, dependent on `ngram`.
-        """
-        n_letters, _ = self._create_n_letters(tokens, ngram)
         return n_letters
 
-    def fit(self, tokens: list, ngram: int=3):
```

Comment:
Is `fit` unwanted? What if I want to fit on training data then use those tri-letters to transform testing data?

Reply:
The best thing to do is to make use of Vocabulary Unit,

Reply:
Make sense.